llama_stack/apis/agents/openai_responses.py

```diff
@@ -1258,9 +1258,9 @@ class OpenAIResponseInputFunctionToolCallOutput(BaseModel):
     | OpenAIResponseInputFunctionToolCallOutput
     | OpenAIResponseMCPApprovalRequest
     | OpenAIResponseMCPApprovalResponse
-    |
-    # Fallback to the generic message type as a last resort
-    OpenAIResponseMessage,
+    | OpenAIResponseOutputMessageMCPCall
```

Comment:
This is similar to the change I'm proposing at https://github.com/llamastack/llama-stack/pull/3385, but there I'm adding diractly OpenAIResponseOutput as a possible input, to always keep both list in sync (based on a comment I received in that PR).

As the other PR is simpler/shorter, perhaps we can go with it and rebase this on top